### ISSUE TITLE
Opiate drugs increase pain

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10117]
**Preferred term label:**
Opiate drugs increase pain

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Narcotic bowel syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000634
- label: Exacerbated by opiate medication
- parents: [Aggravated by exposure to medication (HP:4000176)](https://hpo.jax.org/app/browse/term/HP:4000176)
- pmid: 
- def: Applied to a sign orr symptom that is worsed by administration of opiate medications.
- comment: Opiate drugs increase pain in narcotic bowel syndrome.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10117
- synonyms:
